catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -813,6 +843,179 @@ public void unknownOperation() throws CswException {
         throw new CswException("No such operation", CswConstants.OPERATION_NOT_SUPPORTED, null);
     }
 
+    @Override
+    @DELETE
+    @Path("/subscription/{requestId}")
+    @Produces({MediaType.WILDCARD})
+    public Response deleteRecordsSubscription(@PathParam("requestId") String requestId)
+            throws CswException {
+        CswSubscription subscription = (CswSubscription) SubscriptionManager.getSubscription(
```

Comment:
Since we have a `SubscriptionManager` class, should we just have the `CswEndpoint` class only do input validation and transformation and keep all the business logic down inside the `SubscriptionManager`? That would keep this class a little smaller and would make the `SubscriptionManager` logic more re-usable.